Query Plan Not Working

Issue Type: <b>Bug</b>

Select any query and attempt to show the query plan (Click the Explain button). The Query Plan panel shows up, but is blank.

Errors in the console:
```
log.ts:301   ERR Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.: Error: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.
    at u.onHide (vscode-file://vscode-app/c:/Users/jtsombakos/AppData/Local/Programs/Azure%20Data%20Studio%20-%20Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:3333:27550)
    at m.clearPlans (vscode-file://vscode-app/c:/Users/jtsombakos/AppData/Local/Programs/Azure%20Data%20Studio%20-%20Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:3364:4514)
    at vscode-file://vscode-app/c:/Users/jtsombakos/AppData/Local/Programs/Azure%20Data%20Studio%20-%20Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:3366:11447
    at i.fire (vscode-file://vscode-app/c:/Users/jtsombakos/AppData/Local/Programs/Azure%20Data%20Studio%20-%20Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:129:1712)
    at i.doRunQuery (vscode-file://vscode-app/c:/Users/jtsombakos/AppData/Local/Programs/Azure%20Data%20Studio%20-%20Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:2284:51267)
    at i.runQuery (vscode-file://vscode-app/c:/Users/jtsombakos/AppData/Local/Programs/Azure%20Data%20Studio%20-%20Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:2284:50824)
    at p.doRunQuery (vscode-file://vscode-app/c:/Users/jtsombakos/AppData/Local/Programs/Azure%20Data%20Studio%20-%20Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:2287:3538)
    at p.runQuery (vscode-file://vscode-app/c:/Users/jtsombakos/AppData/Local/Programs/Azure%20Data%20Studio%20-%20Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:2287:3038)
    at Va.runQuery (vscode-file://vscode-app/c:/Users/jtsombakos/AppData/Local/Programs/Azure%20Data%20Studio%20-%20Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:1606:31664)
    at Bs.runQuery (vscode-file://vscode-app/c:/Users/jtsombakos/AppData/Local/Programs/Azure%20Data%20Studio%20-%20Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:2277:6127)
    at Bs.run (vscode-file://vscode-app/c:/Users/jtsombakos/AppData/Local/Programs/Azure%20Data%20Studio%20-%20Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:2277:5939)
    at T.runAction (vscode-file://vscode-app/c:/Users/jtsombakos/AppData/Local/Programs/Azure%20Data%20Studio%20-%20Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:307:30724)
    at T.run (vscode-file://vscode-app/c:/Users/jtsombakos/AppData/Local/Programs/Azure%20Data%20Studio%20-%20Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:307:30627)
    at d.onClick (vscode-file://vscode-app/c:/Users/jtsombakos/AppData/Local/Programs/Azure%20Data%20Studio%20-%20Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:307:36306)
    at Immediate.<anonymous> (vscode-file://vscode-app/c:/Users/jtsombakos/AppData/Local/Programs/Azure%20Data%20Studio%20-%20Insiders/resources/app/out/vs/workbench/workbench.desktop.main.js:307:35861)
    at Immediate.timer (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3083:47)
    at ZoneDelegate.invokeTask (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:434:35)
    at Zone.runTask (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:205:51)
    at ZoneTask.invokeTask (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:516:38)
    at Immediate.ZoneTask.invoke (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:505:52)
    at Immediate.data.args.<computed> [as _onImmediate] (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3063:36)
    at processImmediate (internal/timers.js:461:21)
lifecycle.ts:224 Error: Trying to add a disposable to a DisposableStore that has already been disposed of. The added object will be leaked!
    at g.add (lifecycle.ts:224) [<root>]
    at re._register (lifecycle.ts:255) [<root>]
    at re.build (gridPanel.ts:506) [<root>]
    at re.onDidInsert (gridPanel.ts:420) [<root>]
    at R._runner (scrollableView.ts:282) [<root>]
    at R.execute (dom.ts:210) [<root>]
    at We (dom.ts:250) [<root>]
    at timer (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3083) [<root>]
    at data.args.<computed> (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3063) [<root>]
add @ lifecycle.ts:224
_register @ lifecycle.ts:255
build @ gridPanel.ts:506
onDidInsert @ gridPanel.ts:420
(anonymous) @ scrollableView.ts:282
execute @ dom.ts:210
We @ dom.ts:250
timer @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3083
ZoneDelegate.invokeTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:434
Zone.runTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:205
ZoneTask.invokeTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:516
ZoneTask.invoke @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:505
data.args.<computed> @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3063
requestAnimationFrame (async)
scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3065
ZoneDelegate.scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:421
Zone.scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:248
Zone.scheduleMacroTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:271
scheduleMacroTaskWithCurrentZone @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:716
(anonymous) @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3107
proto.<computed> @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:1015
k @ dom.ts:170
e.scheduleAtNextAnimationFrame @ dom.ts:261
insertItemInDOM @ scrollableView.ts:279
render @ scrollableView.ts:242
rerender @ scrollableView.ts:133
addViews @ scrollableView.ts:146
addResultSet @ gridPanel.ts:244
onResultSet @ gridPanel.ts:159
fire @ event.ts:579
handleResultSetAvailable @ queryRunner.ts:352
(anonymous) @ queryManagement.ts:327
enqueueOrRun @ queryManagement.ts:159
_notify @ queryManagement.ts:165
onResultSetAvailable @ queryManagement.ts:326
$onResultSetAvailable @ mainThreadDataProtocol.ts:585
_doInvokeHandler @ rpcProtocol.ts:473
_invokeHandler @ rpcProtocol.ts:458
_receiveRequest @ rpcProtocol.ts:373
_receiveOneMessage @ rpcProtocol.ts:295
(anonymous) @ rpcProtocol.ts:160
fire @ event.ts:577
fire @ ipc.net.ts:513
_receiveMessage @ ipc.net.ts:866
(anonymous) @ ipc.net.ts:705
fire @ event.ts:577
acceptChunk @ ipc.net.ts:286
(anonymous) @ ipc.net.ts:247
D @ ipc.net.ts:50
emit @ events.js:315
addChunk @ internal/streams/readable.js:309
readableAddChunk @ internal/streams/readable.js:284
Readable.push @ internal/streams/readable.js:223
onStreamRead @ internal/stream_base_commons.js:188
Show 23 more frames
lifecycle.ts:224 Error: Trying to add a disposable to a DisposableStore that has already been disposed of. The added object will be leaked!
    at g.add (lifecycle.ts:224) [<root>]
    at re._register (lifecycle.ts:255) [<root>]
    at re.build (gridPanel.ts:515) [<root>]
    at re.onDidInsert (gridPanel.ts:420) [<root>]
    at R._runner (scrollableView.ts:282) [<root>]
    at R.execute (dom.ts:210) [<root>]
    at We (dom.ts:250) [<root>]
    at timer (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3083) [<root>]
    at data.args.<computed> (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3063) [<root>]
add @ lifecycle.ts:224
_register @ lifecycle.ts:255
build @ gridPanel.ts:515
onDidInsert @ gridPanel.ts:420
(anonymous) @ scrollableView.ts:282
execute @ dom.ts:210
We @ dom.ts:250
timer @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3083
ZoneDelegate.invokeTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:434
Zone.runTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:205
ZoneTask.invokeTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:516
ZoneTask.invoke @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:505
data.args.<computed> @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3063
requestAnimationFrame (async)
scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3065
ZoneDelegate.scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:421
Zone.scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:248
Zone.scheduleMacroTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:271
scheduleMacroTaskWithCurrentZone @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:716
(anonymous) @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3107
proto.<computed> @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:1015
k @ dom.ts:170
e.scheduleAtNextAnimationFrame @ dom.ts:261
insertItemInDOM @ scrollableView.ts:279
render @ scrollableView.ts:242
rerender @ scrollableView.ts:133
addViews @ scrollableView.ts:146
addResultSet @ gridPanel.ts:244
onResultSet @ gridPanel.ts:159
fire @ event.ts:579
handleResultSetAvailable @ queryRunner.ts:352
(anonymous) @ queryManagement.ts:327
enqueueOrRun @ queryManagement.ts:159
_notify @ queryManagement.ts:165
onResultSetAvailable @ queryManagement.ts:326
$onResultSetAvailable @ mainThreadDataProtocol.ts:585
_doInvokeHandler @ rpcProtocol.ts:473
_invokeHandler @ rpcProtocol.ts:458
_receiveRequest @ rpcProtocol.ts:373
_receiveOneMessage @ rpcProtocol.ts:295
(anonymous) @ rpcProtocol.ts:160
fire @ event.ts:577
fire @ ipc.net.ts:513
_receiveMessage @ ipc.net.ts:866
(anonymous) @ ipc.net.ts:705
fire @ event.ts:577
acceptChunk @ ipc.net.ts:286
(anonymous) @ ipc.net.ts:247
D @ ipc.net.ts:50
emit @ events.js:315
addChunk @ internal/streams/readable.js:309
readableAddChunk @ internal/streams/readable.js:284
Readable.push @ internal/streams/readable.js:223
onStreamRead @ internal/stream_base_commons.js:188
Show 23 more frames
lifecycle.ts:224 Error: Trying to add a disposable to a DisposableStore that has already been disposed of. The added object will be leaked!
    at g.add (lifecycle.ts:224) [<root>]
    at re._register (lifecycle.ts:255) [<root>]
    at re.build (gridPanel.ts:516) [<root>]
    at re.onDidInsert (gridPanel.ts:420) [<root>]
    at R._runner (scrollableView.ts:282) [<root>]
    at R.execute (dom.ts:210) [<root>]
    at We (dom.ts:250) [<root>]
    at timer (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3083) [<root>]
    at data.args.<computed> (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3063) [<root>]
add @ lifecycle.ts:224
_register @ lifecycle.ts:255
build @ gridPanel.ts:516
onDidInsert @ gridPanel.ts:420
(anonymous) @ scrollableView.ts:282
execute @ dom.ts:210
We @ dom.ts:250
timer @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3083
ZoneDelegate.invokeTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:434
Zone.runTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:205
ZoneTask.invokeTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:516
ZoneTask.invoke @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:505
data.args.<computed> @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3063
requestAnimationFrame (async)
scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3065
ZoneDelegate.scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:421
Zone.scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:248
Zone.scheduleMacroTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:271
scheduleMacroTaskWithCurrentZone @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:716
(anonymous) @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3107
proto.<computed> @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:1015
k @ dom.ts:170
e.scheduleAtNextAnimationFrame @ dom.ts:261
insertItemInDOM @ scrollableView.ts:279
render @ scrollableView.ts:242
rerender @ scrollableView.ts:133
addViews @ scrollableView.ts:146
addResultSet @ gridPanel.ts:244
onResultSet @ gridPanel.ts:159
fire @ event.ts:579
handleResultSetAvailable @ queryRunner.ts:352
(anonymous) @ queryManagement.ts:327
enqueueOrRun @ queryManagement.ts:159
_notify @ queryManagement.ts:165
onResultSetAvailable @ queryManagement.ts:326
$onResultSetAvailable @ mainThreadDataProtocol.ts:585
_doInvokeHandler @ rpcProtocol.ts:473
_invokeHandler @ rpcProtocol.ts:458
_receiveRequest @ rpcProtocol.ts:373
_receiveOneMessage @ rpcProtocol.ts:295
(anonymous) @ rpcProtocol.ts:160
fire @ event.ts:577
fire @ ipc.net.ts:513
_receiveMessage @ ipc.net.ts:866
(anonymous) @ ipc.net.ts:705
fire @ event.ts:577
acceptChunk @ ipc.net.ts:286
(anonymous) @ ipc.net.ts:247
D @ ipc.net.ts:50
emit @ events.js:315
addChunk @ internal/streams/readable.js:309
readableAddChunk @ internal/streams/readable.js:284
Readable.push @ internal/streams/readable.js:223
onStreamRead @ internal/stream_base_commons.js:188
Show 23 more frames
lifecycle.ts:224 Error: Trying to add a disposable to a DisposableStore that has already been disposed of. The added object will be leaked!
    at g.add (lifecycle.ts:224) [<root>]
    at re._register (lifecycle.ts:255) [<root>]
    at re.build (gridPanel.ts:517) [<root>]
    at re.onDidInsert (gridPanel.ts:420) [<root>]
    at R._runner (scrollableView.ts:282) [<root>]
    at R.execute (dom.ts:210) [<root>]
    at We (dom.ts:250) [<root>]
    at timer (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3083) [<root>]
    at data.args.<computed> (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3063) [<root>]
add @ lifecycle.ts:224
_register @ lifecycle.ts:255
build @ gridPanel.ts:517
onDidInsert @ gridPanel.ts:420
(anonymous) @ scrollableView.ts:282
execute @ dom.ts:210
We @ dom.ts:250
timer @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3083
ZoneDelegate.invokeTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:434
Zone.runTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:205
ZoneTask.invokeTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:516
ZoneTask.invoke @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:505
data.args.<computed> @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3063
requestAnimationFrame (async)
scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3065
ZoneDelegate.scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:421
Zone.scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:248
Zone.scheduleMacroTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:271
scheduleMacroTaskWithCurrentZone @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:716
(anonymous) @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3107
proto.<computed> @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:1015
k @ dom.ts:170
e.scheduleAtNextAnimationFrame @ dom.ts:261
insertItemInDOM @ scrollableView.ts:279
render @ scrollableView.ts:242
rerender @ scrollableView.ts:133
addViews @ scrollableView.ts:146
addResultSet @ gridPanel.ts:244
onResultSet @ gridPanel.ts:159
fire @ event.ts:579
handleResultSetAvailable @ queryRunner.ts:352
(anonymous) @ queryManagement.ts:327
enqueueOrRun @ queryManagement.ts:159
_notify @ queryManagement.ts:165
onResultSetAvailable @ queryManagement.ts:326
$onResultSetAvailable @ mainThreadDataProtocol.ts:585
_doInvokeHandler @ rpcProtocol.ts:473
_invokeHandler @ rpcProtocol.ts:458
_receiveRequest @ rpcProtocol.ts:373
_receiveOneMessage @ rpcProtocol.ts:295
(anonymous) @ rpcProtocol.ts:160
fire @ event.ts:577
fire @ ipc.net.ts:513
_receiveMessage @ ipc.net.ts:866
(anonymous) @ ipc.net.ts:705
fire @ event.ts:577
acceptChunk @ ipc.net.ts:286
(anonymous) @ ipc.net.ts:247
D @ ipc.net.ts:50
emit @ events.js:315
addChunk @ internal/streams/readable.js:309
readableAddChunk @ internal/streams/readable.js:284
Readable.push @ internal/streams/readable.js:223
onStreamRead @ internal/stream_base_commons.js:188
Show 23 more frames
lifecycle.ts:224 Error: Trying to add a disposable to a DisposableStore that has already been disposed of. The added object will be leaked!
    at g.add (lifecycle.ts:224) [<root>]
    at re._register (lifecycle.ts:255) [<root>]
    at re.build (gridPanel.ts:519) [<root>]
    at re.onDidInsert (gridPanel.ts:420) [<root>]
    at R._runner (scrollableView.ts:282) [<root>]
    at R.execute (dom.ts:210) [<root>]
    at We (dom.ts:250) [<root>]
    at timer (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3083) [<root>]
    at data.args.<computed> (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3063) [<root>]
add @ lifecycle.ts:224
_register @ lifecycle.ts:255
build @ gridPanel.ts:519
onDidInsert @ gridPanel.ts:420
(anonymous) @ scrollableView.ts:282
execute @ dom.ts:210
We @ dom.ts:250
timer @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3083
ZoneDelegate.invokeTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:434
Zone.runTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:205
ZoneTask.invokeTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:516
ZoneTask.invoke @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:505
data.args.<computed> @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3063
requestAnimationFrame (async)
scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3065
ZoneDelegate.scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:421
Zone.scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:248
Zone.scheduleMacroTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:271
scheduleMacroTaskWithCurrentZone @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:716
(anonymous) @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3107
proto.<computed> @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:1015
k @ dom.ts:170
e.scheduleAtNextAnimationFrame @ dom.ts:261
insertItemInDOM @ scrollableView.ts:279
render @ scrollableView.ts:242
rerender @ scrollableView.ts:133
addViews @ scrollableView.ts:146
addResultSet @ gridPanel.ts:244
onResultSet @ gridPanel.ts:159
fire @ event.ts:579
handleResultSetAvailable @ queryRunner.ts:352
(anonymous) @ queryManagement.ts:327
enqueueOrRun @ queryManagement.ts:159
_notify @ queryManagement.ts:165
onResultSetAvailable @ queryManagement.ts:326
$onResultSetAvailable @ mainThreadDataProtocol.ts:585
_doInvokeHandler @ rpcProtocol.ts:473
_invokeHandler @ rpcProtocol.ts:458
_receiveRequest @ rpcProtocol.ts:373
_receiveOneMessage @ rpcProtocol.ts:295
(anonymous) @ rpcProtocol.ts:160
fire @ event.ts:577
fire @ ipc.net.ts:513
_receiveMessage @ ipc.net.ts:866
(anonymous) @ ipc.net.ts:705
fire @ event.ts:577
acceptChunk @ ipc.net.ts:286
(anonymous) @ ipc.net.ts:247
D @ ipc.net.ts:50
emit @ events.js:315
addChunk @ internal/streams/readable.js:309
readableAddChunk @ internal/streams/readable.js:284
Readable.push @ internal/streams/readable.js:223
onStreamRead @ internal/stream_base_commons.js:188
Show 23 more frames
lifecycle.ts:224 Error: Trying to add a disposable to a DisposableStore that has already been disposed of. The added object will be leaked!
    at g.add (lifecycle.ts:224) [<root>]
    at re._register (lifecycle.ts:255) [<root>]
    at re.build (gridPanel.ts:520) [<root>]
    at re.onDidInsert (gridPanel.ts:420) [<root>]
    at R._runner (scrollableView.ts:282) [<root>]
    at R.execute (dom.ts:210) [<root>]
    at We (dom.ts:250) [<root>]
    at timer (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3083) [<root>]
    at data.args.<computed> (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3063) [<root>]
add @ lifecycle.ts:224
_register @ lifecycle.ts:255
build @ gridPanel.ts:520
onDidInsert @ gridPanel.ts:420
(anonymous) @ scrollableView.ts:282
execute @ dom.ts:210
We @ dom.ts:250
timer @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3083
ZoneDelegate.invokeTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:434
Zone.runTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:205
ZoneTask.invokeTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:516
ZoneTask.invoke @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:505
data.args.<computed> @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3063
requestAnimationFrame (async)
scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3065
ZoneDelegate.scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:421
Zone.scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:248
Zone.scheduleMacroTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:271
scheduleMacroTaskWithCurrentZone @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:716
(anonymous) @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3107
proto.<computed> @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:1015
k @ dom.ts:170
e.scheduleAtNextAnimationFrame @ dom.ts:261
insertItemInDOM @ scrollableView.ts:279
render @ scrollableView.ts:242
rerender @ scrollableView.ts:133
addViews @ scrollableView.ts:146
addResultSet @ gridPanel.ts:244
onResultSet @ gridPanel.ts:159
fire @ event.ts:579
handleResultSetAvailable @ queryRunner.ts:352
(anonymous) @ queryManagement.ts:327
enqueueOrRun @ queryManagement.ts:159
_notify @ queryManagement.ts:165
onResultSetAvailable @ queryManagement.ts:326
$onResultSetAvailable @ mainThreadDataProtocol.ts:585
_doInvokeHandler @ rpcProtocol.ts:473
_invokeHandler @ rpcProtocol.ts:458
_receiveRequest @ rpcProtocol.ts:373
_receiveOneMessage @ rpcProtocol.ts:295
(anonymous) @ rpcProtocol.ts:160
fire @ event.ts:577
fire @ ipc.net.ts:513
_receiveMessage @ ipc.net.ts:866
(anonymous) @ ipc.net.ts:705
fire @ event.ts:577
acceptChunk @ ipc.net.ts:286
(anonymous) @ ipc.net.ts:247
D @ ipc.net.ts:50
emit @ events.js:315
addChunk @ internal/streams/readable.js:309
readableAddChunk @ internal/streams/readable.js:284
Readable.push @ internal/streams/readable.js:223
onStreamRead @ internal/stream_base_commons.js:188
Show 23 more frames
lifecycle.ts:224 Error: Trying to add a disposable to a DisposableStore that has already been disposed of. The added object will be leaked!
    at g.add (lifecycle.ts:224) [<root>]
    at re._register (lifecycle.ts:255) [<root>]
    at re.build (gridPanel.ts:530) [<root>]
    at re.onDidInsert (gridPanel.ts:420) [<root>]
    at R._runner (scrollableView.ts:282) [<root>]
    at R.execute (dom.ts:210) [<root>]
    at We (dom.ts:250) [<root>]
    at timer (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3083) [<root>]
    at data.args.<computed> (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3063) [<root>]
add @ lifecycle.ts:224
_register @ lifecycle.ts:255
build @ gridPanel.ts:530
onDidInsert @ gridPanel.ts:420
(anonymous) @ scrollableView.ts:282
execute @ dom.ts:210
We @ dom.ts:250
timer @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3083
ZoneDelegate.invokeTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:434
Zone.runTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:205
ZoneTask.invokeTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:516
ZoneTask.invoke @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:505
data.args.<computed> @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3063
requestAnimationFrame (async)
scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3065
ZoneDelegate.scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:421
Zone.scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:248
Zone.scheduleMacroTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:271
scheduleMacroTaskWithCurrentZone @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:716
(anonymous) @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3107
proto.<computed> @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:1015
k @ dom.ts:170
e.scheduleAtNextAnimationFrame @ dom.ts:261
insertItemInDOM @ scrollableView.ts:279
render @ scrollableView.ts:242
rerender @ scrollableView.ts:133
addViews @ scrollableView.ts:146
addResultSet @ gridPanel.ts:244
onResultSet @ gridPanel.ts:159
fire @ event.ts:579
handleResultSetAvailable @ queryRunner.ts:352
(anonymous) @ queryManagement.ts:327
enqueueOrRun @ queryManagement.ts:159
_notify @ queryManagement.ts:165
onResultSetAvailable @ queryManagement.ts:326
$onResultSetAvailable @ mainThreadDataProtocol.ts:585
_doInvokeHandler @ rpcProtocol.ts:473
_invokeHandler @ rpcProtocol.ts:458
_receiveRequest @ rpcProtocol.ts:373
_receiveOneMessage @ rpcProtocol.ts:295
(anonymous) @ rpcProtocol.ts:160
fire @ event.ts:577
fire @ ipc.net.ts:513
_receiveMessage @ ipc.net.ts:866
(anonymous) @ ipc.net.ts:705
fire @ event.ts:577
acceptChunk @ ipc.net.ts:286
(anonymous) @ ipc.net.ts:247
D @ ipc.net.ts:50
emit @ events.js:315
addChunk @ internal/streams/readable.js:309
readableAddChunk @ internal/streams/readable.js:284
Readable.push @ internal/streams/readable.js:223
onStreamRead @ internal/stream_base_commons.js:188
Show 23 more frames
lifecycle.ts:224 Error: Trying to add a disposable to a DisposableStore that has already been disposed of. The added object will be leaked!
    at g.add (lifecycle.ts:224) [<root>]
    at re._register (lifecycle.ts:255) [<root>]
    at re.build (gridPanel.ts:542) [<root>]
    at re.onDidInsert (gridPanel.ts:420) [<root>]
    at R._runner (scrollableView.ts:282) [<root>]
    at R.execute (dom.ts:210) [<root>]
    at We (dom.ts:250) [<root>]
    at timer (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3083) [<root>]
    at data.args.<computed> (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3063) [<root>]
add @ lifecycle.ts:224
_register @ lifecycle.ts:255
build @ gridPanel.ts:542
onDidInsert @ gridPanel.ts:420
(anonymous) @ scrollableView.ts:282
execute @ dom.ts:210
We @ dom.ts:250
timer @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3083
ZoneDelegate.invokeTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:434
Zone.runTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:205
ZoneTask.invokeTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:516
ZoneTask.invoke @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:505
data.args.<computed> @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3063
requestAnimationFrame (async)
scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3065
ZoneDelegate.scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:421
Zone.scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:248
Zone.scheduleMacroTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:271
scheduleMacroTaskWithCurrentZone @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:716
(anonymous) @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3107
proto.<computed> @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:1015
k @ dom.ts:170
e.scheduleAtNextAnimationFrame @ dom.ts:261
insertItemInDOM @ scrollableView.ts:279
render @ scrollableView.ts:242
rerender @ scrollableView.ts:133
addViews @ scrollableView.ts:146
addResultSet @ gridPanel.ts:244
onResultSet @ gridPanel.ts:159
fire @ event.ts:579
handleResultSetAvailable @ queryRunner.ts:352
(anonymous) @ queryManagement.ts:327
enqueueOrRun @ queryManagement.ts:159
_notify @ queryManagement.ts:165
onResultSetAvailable @ queryManagement.ts:326
$onResultSetAvailable @ mainThreadDataProtocol.ts:585
_doInvokeHandler @ rpcProtocol.ts:473
_invokeHandler @ rpcProtocol.ts:458
_receiveRequest @ rpcProtocol.ts:373
_receiveOneMessage @ rpcProtocol.ts:295
(anonymous) @ rpcProtocol.ts:160
fire @ event.ts:577
fire @ ipc.net.ts:513
_receiveMessage @ ipc.net.ts:866
(anonymous) @ ipc.net.ts:705
fire @ event.ts:577
acceptChunk @ ipc.net.ts:286
(anonymous) @ ipc.net.ts:247
D @ ipc.net.ts:50
emit @ events.js:315
addChunk @ internal/streams/readable.js:309
readableAddChunk @ internal/streams/readable.js:284
Readable.push @ internal/streams/readable.js:223
onStreamRead @ internal/stream_base_commons.js:188
Show 23 more frames
lifecycle.ts:224 Error: Trying to add a disposable to a DisposableStore that has already been disposed of. The added object will be leaked!
    at g.add (lifecycle.ts:224) [<root>]
    at re._register (lifecycle.ts:255) [<root>]
    at re.setupState (gridPanel.ts:614) [<root>]
    at re.onDidInsert (gridPanel.ts:435) [<root>]
    at R._runner (scrollableView.ts:282) [<root>]
    at R.execute (dom.ts:210) [<root>]
    at We (dom.ts:250) [<root>]
    at timer (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3083) [<root>]
    at data.args.<computed> (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3063) [<root>]
add @ lifecycle.ts:224
_register @ lifecycle.ts:255
setupState @ gridPanel.ts:614
onDidInsert @ gridPanel.ts:435
(anonymous) @ scrollableView.ts:282
execute @ dom.ts:210
We @ dom.ts:250
timer @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3083
ZoneDelegate.invokeTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:434
Zone.runTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:205
ZoneTask.invokeTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:516
ZoneTask.invoke @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:505
data.args.<computed> @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3063
requestAnimationFrame (async)
scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3065
ZoneDelegate.scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:421
Zone.scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:248
Zone.scheduleMacroTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:271
scheduleMacroTaskWithCurrentZone @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:716
(anonymous) @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3107
proto.<computed> @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:1015
k @ dom.ts:170
e.scheduleAtNextAnimationFrame @ dom.ts:261
insertItemInDOM @ scrollableView.ts:279
render @ scrollableView.ts:242
rerender @ scrollableView.ts:133
addViews @ scrollableView.ts:146
addResultSet @ gridPanel.ts:244
onResultSet @ gridPanel.ts:159
fire @ event.ts:579
handleResultSetAvailable @ queryRunner.ts:352
(anonymous) @ queryManagement.ts:327
enqueueOrRun @ queryManagement.ts:159
_notify @ queryManagement.ts:165
onResultSetAvailable @ queryManagement.ts:326
$onResultSetAvailable @ mainThreadDataProtocol.ts:585
_doInvokeHandler @ rpcProtocol.ts:473
_invokeHandler @ rpcProtocol.ts:458
_receiveRequest @ rpcProtocol.ts:373
_receiveOneMessage @ rpcProtocol.ts:295
(anonymous) @ rpcProtocol.ts:160
fire @ event.ts:577
fire @ ipc.net.ts:513
_receiveMessage @ ipc.net.ts:866
(anonymous) @ ipc.net.ts:705
fire @ event.ts:577
acceptChunk @ ipc.net.ts:286
(anonymous) @ ipc.net.ts:247
D @ ipc.net.ts:50
emit @ events.js:315
addChunk @ internal/streams/readable.js:309
readableAddChunk @ internal/streams/readable.js:284
Readable.push @ internal/streams/readable.js:223
onStreamRead @ internal/stream_base_commons.js:188
Show 23 more frames
lifecycle.ts:224 Error: Trying to add a disposable to a DisposableStore that has already been disposed of. The added object will be leaked!
    at g.add (lifecycle.ts:224) [<root>]
    at re._register (lifecycle.ts:255) [<root>]
    at re.setupState (gridPanel.ts:616) [<root>]
    at re.onDidInsert (gridPanel.ts:435) [<root>]
    at R._runner (scrollableView.ts:282) [<root>]
    at R.execute (dom.ts:210) [<root>]
    at We (dom.ts:250) [<root>]
    at timer (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3083) [<root>]
    at data.args.<computed> (C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3063) [<root>]
add @ lifecycle.ts:224
_register @ lifecycle.ts:255
setupState @ gridPanel.ts:616
onDidInsert @ gridPanel.ts:435
(anonymous) @ scrollableView.ts:282
execute @ dom.ts:210
We @ dom.ts:250
timer @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3083
ZoneDelegate.invokeTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:434
Zone.runTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:205
ZoneTask.invokeTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:516
ZoneTask.invoke @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:505
data.args.<computed> @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3063
requestAnimationFrame (async)
scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3065
ZoneDelegate.scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:421
Zone.scheduleTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:248
Zone.scheduleMacroTask @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:271
scheduleMacroTaskWithCurrentZone @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:716
(anonymous) @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:3107
proto.<computed> @ C:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\node_modules.asar\zone.js\dist\zone.js:1015
k @ dom.ts:170
e.scheduleAtNextAnimationFrame @ dom.ts:261
insertItemInDOM @ scrollableView.ts:279
render @ scrollableView.ts:242
rerender @ scrollableView.ts:133
addViews @ scrollableView.ts:146
addResultSet @ gridPanel.ts:244
onResultSet @ gridPanel.ts:159
fire @ event.ts:579
handleResultSetAvailable @ queryRunner.ts:352
(anonymous) @ queryManagement.ts:327
enqueueOrRun @ queryManagement.ts:159
_notify @ queryManagement.ts:165
onResultSetAvailable @ queryManagement.ts:326
$onResultSetAvailable @ mainThreadDataProtocol.ts:585
_doInvokeHandler @ rpcProtocol.ts:473
_invokeHandler @ rpcProtocol.ts:458
_receiveRequest @ rpcProtocol.ts:373
_receiveOneMessage @ rpcProtocol.ts:295
(anonymous) @ rpcProtocol.ts:160
fire @ event.ts:577
fire @ ipc.net.ts:513
_receiveMessage @ ipc.net.ts:866
(anonymous) @ ipc.net.ts:705
fire @ event.ts:577
acceptChunk @ ipc.net.ts:286
(anonymous) @ ipc.net.ts:247
D @ ipc.net.ts:50
emit @ events.js:315
addChunk @ internal/streams/readable.js:309
readableAddChunk @ internal/streams/readable.js:284
Readable.push @ internal/streams/readable.js:223
onStreamRead @ internal/stream_base_commons.js:188
Show 23 more frames
console.ts:137 [Extension Host] [Error - 2:17:58 PM] Request query/subset failed. (at T.appendLine (c:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\mssql\dist\main.js:2:1690682))
console.ts:137 [Extension Host]   Message: Start row cannot be less than 0 or greater than the number of rows in the result set (Parameter 'startRow')  Code: 0  (at T.appendLine (c:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\mssql\dist\main.js:2:1690682))
console.ts:137 [Extension Host] [Error - 2:17:58 PM] Request query/subset failed. (at T.appendLine (c:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\mssql\dist\main.js:2:1690682))
console.ts:137 [Extension Host]   Message: Start row cannot be less than 0 or greater than the number of rows in the result set (Parameter 'startRow')  Code: 0  (at T.appendLine (c:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\mssql\dist\main.js:2:1690682))
console.ts:137 [Extension Host] [Error - 2:17:58 PM] Request query/subset failed. (at T.appendLine (c:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\mssql\dist\main.js:2:1690682))
console.ts:137 [Extension Host]   Message: Start row cannot be less than 0 or greater than the number of rows in the result set (Parameter 'startRow')  Code: 0  (at T.appendLine (c:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\mssql\dist\main.js:2:1690682))
console.ts:137 [Extension Host] [Error - 2:17:58 PM] Request query/subset failed. (at T.appendLine (c:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\mssql\dist\main.js:2:1690682))
console.ts:137 [Extension Host]   Message: Start row cannot be less than 0 or greater than the number of rows in the result set (Parameter 'startRow')  Code: 0  (at T.appendLine (c:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\mssql\dist\main.js:2:1690682))
console.ts:137 [Extension Host] [Error - 2:17:59 PM] Request query/subset failed. (at T.appendLine (c:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\mssql\dist\main.js:2:1690682))
console.ts:137 [Extension Host]   Message: Start row cannot be less than 0 or greater than the number of rows in the result set (Parameter 'startRow')  Code: 0  (at T.appendLine (c:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\mssql\dist\main.js:2:1690682))
console.ts:137 [Extension Host] [Error - 2:17:59 PM] Request query/subset failed. (at T.appendLine (c:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\mssql\dist\main.js:2:1690682))
console.ts:137 [Extension Host]   Message: Start row cannot be less than 0 or greater than the number of rows in the result set (Parameter 'startRow')  Code: 0  (at T.appendLine (c:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\mssql\dist\main.js:2:1690682))
console.ts:137 [Extension Host] [Error - 2:17:59 PM] Request query/subset failed. (at T.appendLine (c:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\mssql\dist\main.js:2:1690682))
console.ts:137 [Extension Host]   Message: Start row cannot be less than 0 or greater than the number of rows in the result set (Parameter 'startRow')  Code: 0  (at T.appendLine (c:\Users\jtsombakos\AppData\Local\Programs\Azure Data Studio - Insiders\resources\app\extensions\mssql\dist\main.js:2:1690682))
```
Azure Data Studio version: azuredatastudio-insiders 1.36.0-insider (fcd4c6e15bbe7f960c94e4dd1f614298dd18d7ce, 2022-04-13T05:25:49.733Z)
OS version: Windows_NT x64 10.0.19044
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-8500 CPU @ 3.00GHz (6 x 3000)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.78GB (15.42GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (5)</summary>

Extension|Author (truncated)|Version
---|---|---
dacpac|Mic|1.10.0
import|Mic|1.5.0
schema-compare|Mic|1.13.0
sql-database-projects|Mic|0.16.0
sql-search|Red|0.4.0


</details>
<!-- generated by issue reporter -->

--- a/src/sql/workbench/contrib/executionPlan/browser/executionPlanFileView.ts
+++ b/src/sql/workbench/contrib/executionPlan/browser/executionPlanFileView.ts
@@ -41,7 +41,7 @@ export class ExecutionPlanFileView {
 	}
 
 	public onHide(parentContainer: HTMLElement): void {
-		if (parentContainer === this._parent) {
+		if (parentContainer === this._parent && parentContainer.contains(this._container)) {
 			this._parent.removeChild(this._container);
 		}
 	}

--- a/src/sql/workbench/contrib/executionPlan/browser/executionPlanTab.ts
+++ b/src/sql/workbench/contrib/executionPlan/browser/executionPlanTab.ts
@@ -84,6 +84,7 @@ export class ExecutionPlanTabView implements IPanelView {
 			currentView.onHide(this._container);
 			this._input.graphs = [];
 			currentView = this._instantiationService.createInstance(ExecutionPlanFileView);
+			this._viewCache.executionPlanFileViewMap.set(this._input.executionPlanFileViewUUID, currentView);
 			currentView.render(this._container);
 		}
 	}

